tests/json/generator.py

```diff
@@ -53,12 +53,14 @@ class JsonGenerator(object):
         'email': 'email'
     }
 
-    def __init__(self, resolver: RefResolver=None, formats: dict=None) -> None:
+    def __init__(self, resolver: RefResolver=None, formats: dict=None, validate: bool=False) -> None:
```

Comment:
I don't even want this to be optional. Please make the validation mandatory. View it as an important assertion. Correctness always comes first. There is zero value for our team right now in making this optional.